### PR TITLE
Release changelog for 2.27

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+
+  [ Dan Chapman ]
+  * qbs plugin: add plugin support for the Qt Build Suite (qbs). (#1079)
+
+  [ Bayard Randel ]
+  * store: implement delta uploads in push. (#940)
+
+  [ Joe Talbott ]
+  * store: expose store errors to the cli. (#1110)
+  * Make git pulls less error prone due to history changes. (#1052)
```
Comment (elopio) [inline debian/changelog]: you are missing the area:
sources: make git pulls less error prone due to history changes.

**Comment by sergiusens** [comment]
@elopio that is why on Monday I asked you to take care of it :wink:

File: debian/changelog (hunk 1 above)
Comment (sergiusens) [inline debian/changelog]: ah, @kyrofa merged this I guess

Comment (kyrofa) [inline debian/changelog]: Agh, I did, my apologies. I've successfully formed a commit habit for this, but not when hitting the merge button apparently.

**Comment by elopio** [comment]
in the qbs zesty armhf: In file included from /usr/include/arm-linux-gnueabihf/qt5/QtCore/qcoreapplication.h:43:
/usr/include/arm-linux-gnueabihf/qt5/QtCore/qglobal.h:45:12: fatal error: 'cstddef' file not found
#  include <cstddef>

**Pull request merged**